rest/route/middleware.go

```diff
@@ -594,62 +593,35 @@ func updateHostAccessTime(ctx context.Context, h *host.Host) {
 }
 
 func RequiresProjectPermission(permission string, level evergreen.PermissionLevel) gimlet.Middleware {
```

Comment:
We have a few routes in [rest/route/service.go](https://github.com/malikchaya2/evergreen/blob/main/rest/route/service.go#L112)
 that have just viewTasks or which maps to this. With this change, I think we should change them from `( viewTasks)` to `(requireUser, viewTasks)`.

Reply:
viewTasks already does verify user in urlVarsToProjectScopes but can add it more explicitly.